src/main/java/mate/hwdao/controllers/AddManufacturerController.java

```diff
@@ -0,0 +1,36 @@
+package mate.hwdao.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.hwdao.Main;
+import mate.hwdao.lib.Injector;
+import mate.hwdao.model.Manufacturer;
+import mate.hwdao.service.ManufacturerService;
+
+public class AddManufacturerController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance(Main.class.getPackageName());
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/manufacturer/add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        Manufacturer manufacturer = new Manufacturer();
+        String name = request.getParameter("name");
+        String country = request.getParameter("country");
+        manufacturer.setCountry(country);
+        manufacturer.setName(name);
+        manufacturerService.create(manufacturer);
```

Comment:
same here with setters, imho it is not necessary

Reply:
done